packages/core/components/Modal/MoveFileManifest/MoveFileManifest.module.css

```diff
@@ -1,23 +1,43 @@
-.fileListContainer {
-    max-height: 200px;
+.fileTableContainer {
+    max-height: 300px;
     overflow-y: auto;
-    padding: 8px;
     border: 1px solid #ddd;
-    border-radius: 4px;
+    margin-bottom: 1em;
 }
 
-.fileList {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
+.fileTable {
+    width: 100%;
+    border-collapse: collapse;
 }
 
-.fileItem {
-    margin-bottom: 4px;
+.fileTable th,
+.fileTable td {
+    padding: 8px;
+    text-align: left;
+    border-bottom: 1px solid #ddd;
     white-space: nowrap;
 }
 
+.fileTable th {
+    background-color: #f2f2f2;
+    position: sticky;
+    top: 0;
+    z-index: 1;
+}
+
+.fileTable td:first-child {
+    border-right: 1px solid #ddd;
+}
+
+.fileTableContainer::-webkit-scrollbar {
+    width: 8px;
+}
+
+.fileTableContainer::-webkit-scrollbar-thumb {
+    background-color: #888;
+    border-radius: 4px;
+}
 
-.confirmButton {
-    margin-top: 10px;
+.fileTableContainer::-webkit-scrollbar-thumb:hover {
+    background-color: #555;
```

Comment:
I think there's also an example of UX's preferred scrollbar colors in global.css

Reply:
resolved in [588447d](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/588447da65efe53f1b4eebc5459cc97f4b185646)

Reply:
out of curiosity, which browser/version are you using? it's interesting that the global one doesn't seem to get applied in the screenshots

Reply:
The screenshots are from desktop so it should be chrome right?

Reply:
Discussed offline, will be making separate issue